pipelinewise/fastsync/commons/tap_mysql.py

```diff
@@ -207,15 +212,37 @@ def copy_table(self, table_name, path):
                     if not rows:
                         break
 
-
                     # Log export status
                     exported_rows += len(rows)
-                    if (len(rows) == export_batch_rows):
-                        #Then we believe this to be just an interim batch and not the final one so report on progress
-                        utils.log("Exporting batch from {} to {} rows from {}...".format((exported_rows-export_batch_rows),exported_rows, table_name))
+                    if len(rows) == export_batch_rows:
+                        # Then we believe this to be just an interim batch and not the final one so report on progress
+                        utils.log(
+                            "Exporting batch from {} to {} rows from {}...".format((exported_rows - export_batch_rows),
+                                                                                   exported_rows, table_name))
 
                     # Write rows to file
                     for row in rows:
+                        row = list(row)
```

Comment:
If you just need to transform every element of the `row`, you can also simplify this code to one line by using `map()`

Reply:
Not every element, just those of type bytes.

Reply:
`row = list(map(lambda val: codecs.encode(val.decode().rstrip('\x00').encode(), 'hex').decode("utf-8") if isinstance(val, bytes) else val, row))`
Didn't test it but should be smth like this

Reply:
One liners are cool, but do they make an easily readable code?